src/string-ext.ts

```diff
@@ -266,4 +336,14 @@ Object.defineProperty(String.prototype, "hexDecode", {
     {
         return StringExt.hexDecode(this.toString());
     }
+});
+
+Object.defineProperty(String.prototype, "matchesFormat", {
+    configurable: false,
+    enumerable: false,
+    writable: false,
+    value: function (format: string): boolean
+    {
+        return StringExt.matchesFormat(this.toString(), format);
```

Comment:
trim the format before passing it in. We dont want a way to represent whitespace chars at the beginning or end.